bcgov/lucy-web
# [LUCY-1192] FEATURE** Add fields for bilge plugs

> [!NOTE]
> My Linter went wild in these files, sorry in advance for there being 5x as many lines of changes as there should be :)

## Pull Request Standards

- [x] The title of the PR is accurate
- [x] The title includes the type of change [`HOTFIX`, `FEATURE`, `etc`]  
- [x] The PR title includes the ticket number in format of `[LUCY-###]`
- [x] Documentation is updated to reflect change

# Description

This PR includes the following proposed change(s):

- PR Includes a migration to add two new boolean columns to `Watercraft Risk Assessments` (Inspections)
- This is to assist in legislation for BC's [Pull the Plug](https://www2.gov.bc.ca/gov/content/invasive-mussels)
- This is the backend portion of the [iOS Application side](https://github.com/bcgov/invasivesBC-mussels-iOS/pull/312)

## Easiest way to test it

1. Clone this repo
2. Go to Prod or Dev branch
3. From root run `docker-compose -f docker-compose.dev.yml up -d` to make docker images
4. Switch to `this` branch
5. Respin containers and see the migration take place
6. Login to PGAdmin on Port:5050 and see the two new columns appear in the `watercraft_risk_assessment` table

![image](https://github.com/bcgov/lucy-web/assets/62873746/75734704-547e-4c8d-9181-d03ed1c56c95)
